docs/howto.rst

```diff
@@ -125,18 +125,14 @@ Django needs to write its ORM tables:
    ./manage.py migrate
 
 
-Adding new products and releases
+Adding new products
```

Comment:
This is going to interfere with the other PR removing products. If you weren't fond of dealing with git merges like I am, you could remove this whole section in either this PR or the other one so they don't intersect.